Lib/test/test_traceback.py

```diff
@@ -4748,7 +4748,24 @@ class MyList(list):
         with self.assertRaises(TypeError):
             _suggestions._generate_suggestions(MyList(), "")
 
+    def test_no_site_package_flavour(self):
+        code = """import boo"""
+        _, _, stderr = assert_python_failure('-S', '-c', code)
 
+        self.assertIn(
+            (b"Site initialization is disabled, did you forget to "
+                b"add the site-packages directory to sys.path?"), stderr
+        )
+
+        code = """
+            import msvcrt
```

Comment:
This test will fail on Windows, where `msvcrt` *is* available.
Could you either
- go back to testing `boo` after adding it to `stdlib_module_names`, or
- use `grp` on Windows, and `msvcrt` everywhere else?